frontend/src/Utils/Requests.jsx

```diff
@@ -1,54 +1,41 @@
 import axios from "axios";
 
 const sendEmail = emailParameters => {
+  /*
   const emailObject = {
     origin: emailParameters.origin,
     name: emailParameters.name,
     from: emailParameters.from,
     subject: emailParameters.subject,
     body: emailParameters.body,
+    resumeFormat: emailParameters.resumeFormat,
     resumeText: emailParameters.resumeText
-  };
-  axios
-    .post(
-      "/api/email",
-      { emailObject },
-      {
-        headers: {
-          "Content-Type": "application/json"
-        }
+  };*/
```

Comment:
This looks like it is the prod version commented out? If not maybe remove this comment.

Reply:
mmm, didn't intend to have the entire diff in that comment. Ignore the removed code.

Reply:
Yeah, forgot to reset it after finishing testing